src/pages/index.js

```diff
@@ -95,34 +109,111 @@ function handleProfileFormSubmit(inputValues) {
    console.log(err);
   })
     .finally(() => {
-      //profileEditPopup.renderLoading(false,"Save");
+       profileEditPopup.renderLoading("Save");
```

Comment:
Needs correction: you should stop the loading here (now I see `Submitting..` text)